src/WorldStorage/NamespaceSerializer.cpp

```diff
@@ -546,3 +546,87 @@ eMonsterType NamespaceSerializer::ToMonsterType(const std::string_view a_ID)
 
 	return MonsterTypes.at(a_ID);
 }
+
+
+
+
+
+std::string_view NamespaceSerializer::Prettify(const eMonsterType a_ID, const bool a_IsTamed)
```

Comment:
Actually, this meant more universally. The runtime is great, but I meant you provide `some_string` and get `Some String`

Reply:
And yes, this will print ocelot for tamed cats. But this should be fixable by implementing proper cat<->ocelot implementation

Reply:
So you meant that this function should take a string and prettify that input string and not that it constructs a string from a mob ID?

Reply:
Yes 👍

Reply:
That was not that clear from me - sorry